src/tool_launcher/app.config

```diff
@@ -0,0 +1,18 @@
+<!--
+Generated by the .NET Core Command Line.
+-->
+<configuration>
+  <startup>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.2" />
```

Comment:
need to be 3.5 to support windows 7

Reply:
Hmm, that will actually cause a problem for newer computers. .NET Fx 3.5 is not installed by default on Windows 10. Is it safe to say we require .NET Framework 4.5 or higher to make this feature work? i.e. Visual Studio 2017 requires .NET Framework 4.5